composer.json

```diff
@@ -7,6 +7,15 @@
     "prefer-stable": true,
     "minimum-stability": "dev",
     "require": {
-        "php": ">=7.0"
-    }
+        "php": ">=7.0",
+        "adci/full-name-parser": "^0.2.4",
+        "ramsey/uuid": "^3.7",
+        "swedbank-pay/swedbank-pay-woocommerce-core": "dev-master"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/SwedbankPay/swedbank-pay-woocommerce-core"
```

Comment:
Since [`swedbank-pay-woocommerce-core` is published on Packagsit](https://packagist.org/packages/swedbank-pay/swedbank-pay-woocommerce-core), we shouldn't need this `vcs` repository. We can also make a release and tag for it so we don't have to use `dev-master`.